lib/index.js

```diff
@@ -681,6 +844,52 @@ MaplibreGeocoder.prototype = {
     return request;
   },
 
+  _getSuggestions: function (searchInput) {
+    return new Promise(
+      function (resolve, reject) {
```

Comment:
For my understanding, why `function` instead of arrow functions here? Arrow functions could remove the need for `.bind`s.

Reply:
This library is not set up for ES6 code at this moment. I'm actually refactoring the few I used in this PR to ES5 functions right now. 😅